packages/nodes-base/nodes/Form/utils.ts

```diff
@@ -48,8 +53,11 @@ function sanitizeHtml(text: string) {
 		],
 		allowedAttributes: {
 			a: ['href', 'target', 'rel'],
+			img: ['src', 'alt', 'width', 'height'],
+			video: ['*'],
+			iframe: ['*'],
```

Comment:
Should we set some security related attributes on iframes? Thinking of [`sandbox`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox), `allow` and `referrerpolicy`. You could set it using a [transformer](https://github.com/apostrophecms/sanitize-html/blob/main/README.md#transformations) from `sanitize-html`.